tests/queries/0_stateless/02932_kill_query_sleep.sh

```diff
@@ -8,18 +8,31 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 function wait_query_started()
 {
     local query_id="$1"
-    $CLICKHOUSE_CLIENT --query "SYSTEM FLUSH LOGS"
-    while [[ $($CLICKHOUSE_CLIENT --query="SELECT count() FROM system.query_log WHERE query_id='$query_id' AND current_database = currentDatabase()") == 0 ]]; do
```

Comment:
I've changed the code so now `system.processes` is checked here instead of `system.query_log` and also timeouts are used.